### PR TITLE
Updated speakers.json

### PR DESCRIPTION
**Pull request by kominoshja**
added Redon, Elio, Kristi, Tanay, Riccardo and myself

### CODE REVIEW

File: speakers.json [hunk 1]
```diff
@@ -273,6 +273,54 @@
...
+		      "url": "https://mozillians.org/en-US/u/kristi.progri",
+		      "gravatar": "kristiprogri94@gmail.com",
+		      "country": "Albania",
+		      "lat": 41.328889,
+		      "lng": 19.817778
+		},
+			{
+		      "name": "Tanay Pant", 
+		      "url": "https://mozillians.org/en-US/u/tanay1337",
+		      "gravatar": "btanay1337@gmail.com",
```
Comment (tanay1337) [inline speakers.json]: This should be `tanay1337@gmail.com` :)

**Comment by kominoshja** [comment]
Sorry, fixing it nowOn Jul 1, 2017 16:24, Tanay Pant <notifications@github.com> wrote:@tanay1337 commented on this pull request.



In speakers.json:
> +		      "country": "Albania",
+		      "lat": 41.328889,
+		      "lng": 19.817778
+		},
+			{
+		      "name": "Kristi Progri", 
+		      "url": "https://mozillians.org/en-US/u/kristi.progri",
+		      "gravatar": "kristiprogri94@gmail.com",
+		      "country": "Albania",
+		      "lat": 41.328889,
+		      "lng": 19.817778
+		},
+			{
+		      "name": "Tanay Pant", 
+		      "url": "https://mozillians.org/en-US/u/tanay1337",
+		      "gravatar": "btanay1337@gmail.com",

This should be tanay1337@gmail.com :)

File: speakers.json [hunk 2]
```diff
@@ -307,7 +307,7 @@
 			{
 		      "name": "Tanay Pant", 
 		      "url": "https://mozillians.org/en-US/u/tanay1337",
-		      "gravatar": "btanay1337@gmail.com",
+		      "gravatar": "tanay1337@gmail.com",
 		      "country": "Berlin",
```
Comment (tanay1337) [inline speakers.json]: Country: "Germany" :)

**Comment by tanay1337** [comment]
Thanks @Borisbudini :D

**Comment by kominoshja** [comment]
I'm sorry for the multiple problems with your entry 😌🎈On Jul 1, 2017 18:14, Tanay Pant <notifications@github.com> wrote:Thanks @Borisbudini :D

**Comment by tanay1337** [comment]
Not a problem @Borisbudini :D

**Pull request merged**